### DISCUSSION TITLE
Error while trying to install nuclei: proxy.golang.org TLS handshake timeout

### DISCUSSION THREAD

**Discussion by shadihh9** [projectdiscovery/nuclei#1045] [Q&A] [answered]
go get: module github.com/projectdiscovery/nuclei/v2/cmd/nuclei: Get "https://proxy.golang.org/github.com/projectdiscovery/nuclei/v2/cmd/nuclei/@v/list": net/http: TLS handshake timeout
i see this problem in vps

**Comment by geeknik** [comment, marked as answer]
Hello, from the information provided this sounds like your VPS is having issues talking to proxy.golang.org. Could be for any number of reasons: proxy.golang.org is down; your IP is blocked; etc. Not much anyone here can do to resolve this. I would suggest trying from another IP if possible as a first step.
